### PR TITLE
New Port: Pachi el Marciano

### PR DESCRIPTION
**Pull request by Cebion**
Game Information
Title: Pachi el Marciano
URL: https://sourceforge.net/projects/dragontech/

| Button | Action |
|--|--| 
|DPAD| Move|
|A | Jump|

CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] ROCKNIX
[X] MuOS
[X] Knulli (Optional)
[] Crossmix (Optional)

Architectures: 
[x] Aarch64
[X] Panfrost (Mainline)

Resolutions: 
[] 480x320 (Optional)
[X] 640x480
[] 720x720 (RGB30) (Optional)
[X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/pachi/gameinfo.xml
Comment (tabreturn) [inline ports/pachi/gameinfo.xml:1]: shouldn't it be? when this is made --> when this is met

File: ports/pachi/port.json
Comment (tabreturn) [inline ports/pachi/port.json:1]: - Remove the \n symbols in the desc? These seem oddly placed.
- Pachi el marciano is a platforms game --> Pachi el marciano is a platform game
- when this is\nmade --> when this is met

**Review by tabreturn — COMMENTED** [review]
See comments

**Review by cdeletre — DISMISSED** [review]

**Review by tabreturn — APPROVED** [review]

File: ports/pachi/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Pachi el Marciano</path>
+    <name>Pachi el Marciano</name>
+    <desc>Pachi el marciano is a platforms game inspired by games like Manic Miner and
+Jet Set Willy from the legend Mathew Smith.
+
+The goal of the game is to collect all the objects of each level, when this is
+met, the exit gate to the next stage will appear and you can pass trough.</desc>
```
Comment (kloptops) [inline ports/pachi/gameinfo.xml:10]:
```suggestion
met, the exit gate to the next stage will appear and you can pass through.</desc>
```

File: ports/pachi/port.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "Pachi el Marciano.sh",
+    "pachi"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Pachi el Marciano",
+    "porter": [
+      "Cebion"
+    ],
+    "desc": "Pachi el marciano is a platform game inspired by games like Manic Miner and Jet Set Willy from the legend Mathew Smith. The goal of the game is to collect all the objects of each level, when this is met, the exit gate to the next stage will appear and you can pass trough.",
```
Comment (kloptops) [inline ports/pachi/port.json:14]:
```suggestion
    "desc": "Pachi el marciano is a platform game inspired by games like Manic Miner and Jet Set Willy from the legend Mathew Smith. The goal of the game is to collect all the objects of each level, when this is met, the exit gate to the next stage will appear and you can pass through.",
```

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by kloptops — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**